### DISCUSSION TITLE
How to deploy turborepo kitchen-sink example express app in vercel?

### DISCUSSION THREAD

**Discussion by jagadeeshpalaniappan** [vercel/turborepo#1479] [Help] [answered]
How to deploy turborepo kitchen-sink example `express` api app in vercel?

https://github.com/vercel/turborepo/tree/main/examples/kitchen-sink/apps/api

Root Dir: https://github.com/vercel/turborepo/tree/main/examples/kitchen-sink/apps/api
I overridden build command as `cd ../.. && yarn run build --filter=api` . I am still getting error. Could you please help

**Comment by anthonyshew** [comment, marked as answer]
Deploying long-running `express` applications is not available on Vercel today.
